tests-legacy/modules/Invoice/ServiceTest.php

```diff
@@ -113,10 +113,10 @@ public static function dataForSearchQuery()
                 ],
             ],
             [
-                ['paid_at' => 1_353_715_200],
+                ['paid_at' => '1353715200'],
```

Comment:
I think our PHP-CS rules will automatically revert this back to the other style using underscores as separators

Reply:
> actually requires a string

Oh, that's probably why it didn't change it back automatically too. Since it's now a string with your change rather than a literal number.

> This began to cause errors when declare(strict_types=1); was added.

On a similar note, adding `declare(strict_types=1);` can be very risky with the older code. A lot of this was written well before PHP even had strong types at all and relied on types not being strict.. Just some food for thought since I do know you've added some.

I know in the past it's caused breakages due to improper testing not catching the dependency on loose type